deps.edn

```diff
@@ -1,14 +1,16 @@
 {:paths ["components/src" "src" "react-native/src/cljsjs" "resources"]
  :deps  {org.clojure/clojure         {:mvn/version "1.10.0"} ;; Keep in sync with version from Nix
          org.clojure/clojurescript   {:mvn/version "1.10.520"}
+         com.google.javascript/closure-compiler-unshaded {:mvn/version "v20190325"}
```

Comment:
could you please elaborate on these two lines

Reply:
I added these lines to make it the same as in project.clj. If it doesn't make sense I can remove them (at the time I was considering using a plugin to allow us to use the edn file in project.clj).

Reply:
Actually, I'll just get rid of them, since they're not required in this PR. Thanks for pointing them out.